### DISCUSSION TITLE
Texture blending - Very hard edges on my channel mapped textures

### DISCUSSION THREAD

**Discussion by Satchelmouth** [TokisanGames/Terrain3D#269] [Questions]
I've followed the instructions for creating channel mapped textures [here](https://terrain3d.readthedocs.io/en/latest/docs/textures.html), but my textures seem to always come out with extremely hard edges / ignore opacity. Is there any reason for this someone can explain? 
![image](https://github.com/TokisanGames/Terrain3D/assets/118391323/09c58e3c-4f69-4306-a4b1-318137b67597)

**Comment by Satchelmouth** [comment]
Interestingly this only seems to happen on my terrain, not on the flat terrain (where the textures blend correctly

**Comment by TokisanGames** [comment]
Need version info. 
What is "your terrain" as opposed to "flat terrain"?
Use the recommended painting technique in the documentation, Tips. 
Paint in the demo where the project is setup properly. If it works, compare your settings vs the demo.

**Reply by TokisanGames** [reply]
With the release of both tutorial videos, I've now fully explained how to setup textures and paint to get the best results with smooth blending. Also with no further response, I'll assume there's no further issue.
